src/react/components/StyleEditor/Preview.jsx

```diff
@@ -110,7 +110,9 @@ export class Preview extends React.Component {
         
         this.previewAttributes.strokeLineCap = format === 'line' ? this.props.oskariStyle.stroke.lineCap : defaults.stroke.lineCap;
 
-        this.previewAttributes.strokeDashArray = format !== 'point' && this.props.oskariStyle.stroke.lineDash === 'dash' ? '4, 4' : '';
+        this.previewAttributes.strokeDashArray = format === 'line' && this.props.oskariStyle.stroke.lineDash === 'dash' ? '4, 4' // line tab
+            : format === 'area' && this.props.oskariStyle.stroke.area.lineDash === 'dash' ? '4, 4' // area tab
+            : ''; // point tab
```

Comment:
Simplify this. Move it to a function with multiple return lines or something but don't try single-lining it :) It's hard to read == not maintainable.